app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+
+    cd, file_name, new_file = command.split(" ")
```

Comment:
Before better to check that command.split() have exactly 3 parameters)
Also no need to write: command.split(" ") space is default. Just use command.split()